### DISCUSSION TITLE
Docs improvement: install permissions

### DISCUSSION THREAD

**Discussion by danshearer** [Leantime/leantime#578] [General]
(it seems better to use Discussions first, rather than raising an issue)

The directory 'userfiles' needs write permission by the user the webserver is running as, eg www-data. Otherwise operations such as changing the profile picture fail with an error in the errorlog and the app returns to the authentication screen.

For the stable distribution file, it would be a good idea to include an installation step of "chown -R root:www-data" or equivalent, rather than continuing with the id in the tar file, which could map to anything on the user's server.

**Comment by nuvolettablu** [comment]
Hello
I installed the latest version on a shared hosting environment.

Unfortunately, I’m unable to display the profile image.

The upload process works correctly (the photos are successfully stored in the USERFILES folder), but they are not being displayed on the website.

The company logo works (it is shown correctly)
What could I do to solve this issue?

Thank you.
